### ISSUE TITLE
Linux release zip: files not nested in a folder

### ISSUE THREAD

**Issue by jacobcooper** [theotherp/nzbhydra2#288]
The contents of the linux release zip files will extract to the current working directory (same location as the zip), which makes a huge mess. How can we change the release builds to nest them in a folder, preferably with a name that contains the release version?

Apologies for not opening a PR ; I am currently unfamiliar with your build pipeline.

https://github.com/theotherp/nzbhydra2/releases/download/v2.2.1/nzbhydra2-2.2.1-linux.zip
```
$ unzip nzbhydra2-2.2.1-linux.zip 
Archive:  nzbhydra2-2.2.1-linux.zip
   creating: lib/
  inflating: changelog.md            
  inflating: lib/core-2.2.1-exec.jar  
   creating: rc.d/
  inflating: LICENSE                 
   creating: systemd/
  inflating: readme.md               
   creating: sysv/
   creating: upstart/
  inflating: nzbhydra2               
  inflating: rc.d/nzbhydra2          
  inflating: systemd/nzbhydra2.service  
  inflating: sysv/sysv               
  inflating: upstart/defaults        
  inflating: upstart/init.ubuntu  
```

**Comment by theotherp** [comment]
Why?

**Comment by danielb2** [comment]
This is standard practice. To tell you why, I just unzipped this in `/usr/local/` expecting it to actually honor the convention of creating a folder. It did not. It splattered files everywhere. So I started deleting. Including deleting `/usr/local/lib` to which the unzip also unpacked in. Big mistake. now I have to recompile a ton of packages.  NOT COOL.

PLEASE follow convention.

**Comment by danielb2** [comment]
If you think this is just a matter of taste, please look at how other packages do this. I can't tell you how frustrating this is for me right now.

**Comment by theotherp** [comment]
Closed because I've already changed it but haven't released it.

What donation button?

**Comment by danielb2** [comment]
the paypal link doesn't work here:

<img width="485" alt="Screen Shot 2020-05-10 at 14 11 51" src="https://user-images.githubusercontent.com/26678/81510640-3778ce80-92c8-11ea-8ba2-77b7f354f86b.png">

**Comment by theotherp** [comment]
I don't even know where that comes from. Just send money via paypal to that address.

The thing is, it's not about just changing the content of the zip and you're done. All non-dockerized linux instances out there download the zip for the automatic update. Any change to the zip structure kills the update. There are multiple docker container builds out there which extract the zip and their builds would fail if I just changed the zip.

I understand your frustration but this isn't quickly done. I closed the ticket because I already have a solution in the pipeline but it just takes time.
